app/app.go

```diff
@@ -1130,7 +1130,7 @@ func (app *App) ProcessBlockConcurrent(
 }
 
 func (app *App) ProcessTxs(
-	ctx sdk.Context,
+	ctx *sdk.Context,
```

Comment:
why does this need to be a pointer? ctx should be immutable

Reply:
because we modify the context with the list of bad wasm dependencies (previously this was done inline but now that this is in a helper function we need to persist the context changes back so that the context is updated for endblocker